packages/storybook/src/documentation/pages/demos/pagination/serverPagination.demo.tsx

```diff
@@ -0,0 +1,74 @@
+import { Columns, FeatureMode, PageChangedParams, RowsProp, XGrid } from '@material-ui/x-grid';
+import { useCallback, useState } from 'react';
```

Comment:
```suggestion
```

Reply:
auto import, using alt enter ;)

Reply:
One advantage of import *, you don't even need auto import 🙌.

Reply:
lol I love importing what I need only, using my keyboard shortcuts, one advantage of webstorm. I'm sure there is a vscode plugin for that :D